api/src/test/java/org/apache/iceberg/expressions/TestEvaluatior.java

```diff
@@ -156,11 +156,12 @@ public void testCaseInsensitiveNot() {
   @Test
   public void testCaseSensitiveNot() {
     TestHelpers.assertThrows(
-      "X != x when case sensitivity is on",
-      ValidationException.class,
-      "Cannot find field 'X' in struct",
-      () -> { new Evaluator(STRUCT, not(equal("X", 7)), true); }
-      );
+        "X != x when case sensitivity is on",
+        ValidationException.class,
+        "Cannot find field 'X' in struct",
+        () -> {
```

Comment:
Does this need to be a block or can it be an expression?

Reply:
Expression's a great idea.